src/main/java/team/baymax/logic/commands/patient/FindPatientCommand.java

```diff
@@ -15,7 +15,7 @@
  */
 public class FindPatientCommand extends Command {
 
-    public static final String COMMAND_WORD = "findpatient";
+    public static final String COMMAND_WORD = "findpat";
```

Comment:
shotening to "pat" from "patient" could be confusing for the user as "pat" is not a conventional short form for patient (unlike appt)